reference/interbit-ui-components/CONTRIBUTING.adoc

```diff
@@ -0,0 +1,205 @@
+= Contributing
+
+This document is for developers who wish to contribute to the
+`interbit-ui-components` package within the `interbit` repo. For general
+information on contributing to
+link:https://github.com/interbit/interbit[Interbit], please see this
+link:https://github.com/interbit/interbit/blob/master/CONTRIBUTING.md[Contribution Guide].
+
+
+== Creating components
+
+=== File structure
+
+[cols="1a,1a", options="header"]
```

Comment:
When clicking 'View' in Github, this table does not display correctly. Lines are truncated and everything from 'Code Structure' onwards is in the right hand column in the table.

Reply:
I've verified that the table is rendering properly, both on GitHub and in the rendered GitBook HTML.